components/telemetry-operator/internal/webhook/logpipeline_webhook.go

```diff
@@ -236,3 +245,18 @@ func (v *LogPipelineValidator) validateSecrets(ctx context.Context, logPipeline
 	secVal := secret.NewSecretHelper(v.Client)
 	return secVal.ValidateSecretsExist(ctx, logPipeline)
 }
+
+func (v *LogPipelineValidator) validateOutput(logPipeline *telemetryv1alpha1.LogPipeline) error {
```

Comment:
Let's move this to a OutputValidator. Then also add unit tests for it and cover it in the logpipeline_webhook_test.